samples/system-test/automlTranslation.test.js

```diff
@@ -0,0 +1,147 @@
+/**
+ * Copyright 2018, Google, LLC.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* eslint node/no-extraneous-require: off */
+/* eslint node/no-unsupported-features/es-syntax: off */
+
+'use strict';
+
+const test = require(`ava`);
+const tools = require(`@google-cloud/nodejs-repo-tools`);
+
+const cmdDataset = `node ./automl/automlTranslationDataset.js`;
+const cmdModel = `node ./automl/automlTranslationModel.js`;
+const cmdPredict = `node ./automl/automlTranslationPredict.js`;
+
+// const flowerTrainData = `gs://`;
+// const cwd = path.join(__dirname, `..`);
+
+const testDataSetName = `testDataSet`;
+const dummyDataSet = `dummyDataSet`;
+const testModelName = 'dummyModel';
+const sampleText = `./automl/resources/testInput.txt`;
+
+const donotdeleteModelId = `TRL188026453969732486`;
```

Comment:
If either of the following is an issue, we might want to consider making these environment variables.
a) I assume it's OK if these are public?
b) Can this test be run without access to your project, or will these values need to be changed?

Reply:
These are in gs://nodejs-docs-samples-vcm/ so I assumed they're publicly accessible (may need some clarification on this @beccca ) - these are for tests that show predict using premade model to avoid 15+ minute build time for models- this matches convention on cannonicals here: https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/translate/automl/predict_test.py